copyright

```diff
@@ -1459,6 +1453,13 @@ Files:
 Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
 License: CC-BY-SA-4.0
 
+Files:
+ images/outfit/small?photovoltaic*
+ images/outfit/tiny?photovoltaic*
+Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
+License: CC-BY-SA-4.0
```

Comment:
Old license is 3.0, are we allowed to make the new one 4.0?

Reply:
Idk honestly.

Reply:
If I'm reading [this](https://wiki.creativecommons.org/wiki/4.0_upgrade_guidelines) correctly then you can use 4.0 in a context where 3.0 was used previously. Not sure what the benefits are though.